eidas-saml-parser/src/dist/config.yml

```diff
@@ -8,10 +8,16 @@ server:
 
 logging:
   level: INFO
-  loggers:
-    uk.gov.ida: DEBUG
   appenders:
     - type: console
+      layout:
+        type: json
```

Comment:
any reason to use a different lib for these logs? could use logstash-console from dropwizard-logstash to match the other logging outputs

dropwizard-logstash is set up so the logs have standardised keys in our logging saas

Reply:
Wanted to output to StdOut in JSON format rather than multilined. I can still use logstash-console and that should keep the standardised keys. logstash-file will output the logs to json in a file but unable to configured StdOut to do so without the dropwizard-json lib.